source/Symbol/SwiftASTContext.cpp

```diff
@@ -1765,58 +1765,6 @@ lldb::TypeSystemSP SwiftASTContext::CreateInstance(lldb::LanguageType language,
         swift_ast_sp->SetPlatformSDKPath(serialized_sdk_path);
       }
     }
-
-    if (!got_serialized_options || swift_ast_sp->GetPlatformSDKPath().empty()) {
```

Comment:
We need to at the very least log an error to the types log if (!got_serialized_options). It shouldn't be shown to the user necessarily because they may have compiled without serialized debug options.

Reply:
Fair.

Reply:
That's more a question than a request...